dali/benchmark/resnet50_nvjpeg_bench.cc

```diff
@@ -112,7 +112,7 @@ BENCHMARK_DEFINE_F(RealRN50, nvjpegPipe)(benchmark::State& st) { // NOLINT
     }
   }
 
-  WriteHWCBatch<uint8_t>(*ws.Output<GPUBackend>(0), 0, 1, "img");
+//  WriteHWCBatch(*ws.Output<GPUBackend>(0), "img");
```

Comment:
IIRC, the reason we keep this line here is for debugging, right?  If so can we put it in `#if DALI_DEBUG` instead of commenting it out?  Like was done below in dali/pipeline/operators/fused/crop_mirror_normalize_test.cc .

Reply:
DONE